codepropertygraph/src/main/scala/io/shiftleft/Implicits.scala

```diff
@@ -2,6 +2,8 @@ package io.shiftleft
 
 object Implicits {
 
+  class NoSuchNodeException extends RuntimeException
+
   /**
     * A wrapper around a Java iterator that throws a proper NoSuchElementException.
```

Comment:
Please update comment according to change.

Reply:
Done